src/Bundle/test/src/Repository/BookRepository.php

```diff
@@ -26,4 +27,12 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Book::class);
     }
+
+    public function createListQueryBuilder(): QueryBuilder
+    {
+        $queryBuilder = $this->createQueryBuilder('o');
+        $queryBuilder->andWhere($queryBuilder->expr()->isNotNull('o.archivedAt'));
+
+        return $queryBuilder;
+    }
```

Comment:
Why did you add this code?

Reply:
The goal of this PR is to test if the grid configuration is applied (such as a repository method defined on the grid)

Reply:
ok, but where is it used?